### ISSUE TITLE
[RFE] Badge for build status

### ISSUE THREAD

**Issue by blaisep** [darrylwest/pydomkeys#1]
Background: the main branch should always be green. The status of the last build is displayed with a badge that changes color based on condition. Badges improve confidence and summarize information about the project.

Scenario: CI runs after a merge
Given: Build failure
When: User visits the repo
Then: the badge displays red

**Comment by darrylwest** [comment]
yes, informative badges like build status, versioning, etc are important.  I have them on some of my other projects but haven't configured Actions for this project yet.

**Comment by blaisep** [comment]
for inspiration: https://github.com/action-badges/poetry-badges
